charts/opensrp-web/values.yaml

```diff
@@ -186,6 +187,21 @@ express:
   sessionPath: "/"
   reactBuildPath: "/usr/src/web"
   nodeEnv: "production"
+  csp:
+    report-uri: "https://{{ .Values.sentry.domain }}/api/{{ .Values.sentry.projectId }}/security/?sentry_key={{ .Values.sentry.sentryKey }}&sentry_environment={{ .Values.sentry.environment }}&sentry_release={{ .Release.Name }}"
```

Comment:
also add the sentry dns to `connect-src`
![Screenshot from 2022-05-19 14-54-52](https://user-images.githubusercontent.com/10197807/169287550-21ebb562-49c2-4cf7-ba85-2f6eedd90a6e.png)